Linux/powerpc does not like use of fprintf without stdio include

Index: desktop/source/migration/services/jvmfwk.cxx
===================================================================
RCS file: /cvs/framework/desktop/source/migration/services/jvmfwk.cxx,v
retrieving revision 1.4
diff -u -3 -p -r1.4 jvmfwk.cxx
--- desktop/source/migration/services/jvmfwk.cxx	27 Jan 2005 14:27:09 -0000	1.4
+++ desktop/source/migration/services/jvmfwk.cxx	16 Feb 2005 18:50:57 -0000
@@ -79,6 +79,7 @@
 #include "jvmfwk/framework.h"
 #include "jvmfwk.hxx"
 #include <stack>
+#include <stdio.h>
 
 #include "osl/thread.hxx"
 #define OUSTR(x) rtl::OUString(RTL_CONSTASCII_USTRINGPARAM( x ))
